src/components/vmd-commune-or-departement-selector.component.ts

```diff
@@ -134,25 +134,31 @@ export class VmdCommuneOrDepartmentSelectorComponent extends LitElement {
 
     render() {
         return html`
-          <form class="autocomplete ${classMap({'_open': this.showDropdown, '_withButton': this.filter !== ''})}"
+          <form class="row align-items-center"
                 @submit="${this.handleSubmit}">
-            <input type="search" class="autocomplete-input"
-                   required
-                   @focusin="${() => { this.inputHasFocus = true; window.scroll({ top: this.offsetTop - 32, behavior: 'smooth' }); }}"
-                   @focusout="${this.hideDropdownWhenInputHasNotFocus}"
-                   @keydown="${this.handleKeydown}"
-                   @keyup="${this.valueChanged}"
-                   .value="${this.filter}"
-                   placeholder="Commune, Code postal, Département..."
-            />
-            ${this.filter?html`
-            <button type="button" class="autocomplete-button" @click="${() => { this.filter = ''; this.shadowRoot!.querySelector("input")!.focus(); } }"><span>${SVG_CLOSE_ICON}</span></button>
-            `:html``}
-            ${this.recuperationCommunesEnCours?html`
-              <div class="spinner-border text-primary" style="height: 25px; width: 25px" role="status">
-              </div>
-            `:html``}
-            ${this.showDropdown?html`<ul class="autocomplete-results">${this.renderListItems()}</ul>`:html``}
+            <label for="searchAppointment-searchbar" class="col-sm-24 col-md-auto mb-md-1 label-for-search p-3">
```

Comment:
maintenant c'est le label qui a trop de padding-left :)
![image](https://user-images.githubusercontent.com/1426314/117580063-46ba7400-b0f6-11eb-9e70-6b4004dc03e5.png)

 
```suggestion
            <label for="searchAppointment-searchbar" class="col-sm-24 col-md-auto mb-md-1 label-for-search p-3 ps-0">
```

Reply:
Aie, me suis fait avoir avec le dernier commit. Je corrige !
Merci 🙂

Reply:
Le résultat: <img width="576" alt="Capture d’écran 2021-05-09 à 18 48 36" src="https://user-images.githubusercontent.com/5034521/117580210-2c34ca80-b0f7-11eb-8ce8-75b6cd03fe23.png">